contracts/external/cw-vesting/src/msg.rs

```diff
@@ -193,6 +194,27 @@ pub enum QueryMsg {
     Distributable {
         /// The time, as a unix timestamp in seconds, or none to use
```

Comment:
```suggestion
        /// The time, as a unix timestamp in nanoseconds, or none to use
```

comment needs updating. i think `Timestamp`s expect `nanoseconds`

Reply:
thanks!